.github/workflows/release.yaml

```diff
@@ -0,0 +1,70 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      ghcr_image_repository:
+        description: The image repository on GitHub Container Registry.
+        required: true
+        default: cfcontainerizationbot/kubecf-kubectl
+      kubectl_version:
+        description: The kubectl version to build.
+        required: true
+        default: v1.18.9
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Draft release
+      id: draft_release
+      uses: actions/create-release@v1
```

Comment:
I'm not sure what the point of the release is? As far as I can tell, the release system is completely disconnected from the packages system, so we essentially just get a tag with no useful code (it's in kubernetes.git), no useful artifacts (it's under packages), and no useful comments (it's basically hard-coded).

The image metadata already contains the revision used to build this image.

Reply:
Anyone coming to this repository can quickly find the images published to the packages. I'll add the tag to the comments.

I don't see why not publish these releases. :)